app/assets/json/translations/react/en.json

```diff
@@ -1563,6 +1563,7 @@
   "shortFormNav.review": "Review",
   "shortFormNav.you": "You",
   "signIn.badCredentials": "Invalid login credentials. Please try again.",
+  "signIn.badCredentialsWithResetLink": "Email or password is incorrect. Check for mistakes and try again. <a target='_blank' href='%{url}'>Reset your password</a> if you need to.",
```

Comment:
Do we have confirmation from design that we want this to open in a new window/tab?

Reply:
I will ask!

Reply:
will change to same tab